### PR TITLE
Adopt ErrorContent for federation user on community

### PR DESCRIPTION
**Pull request by Sergey-weber**
Add prop isHtmlInDescription for ErrorContent
Hide navigation for federation user on community

### CODE REVIEW

File: src/ui/store/reducers/asideHeader.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function getLandingNavigationSettings() {
 
 const initialSettings: AsideHeaderSettings = {
     withHelpCenter: true,
-    withNavigation: true,
+    withNavigation: !DL.USER.isFederationUserOnCommunity,
```
Comment (imsitnikov) [inline src/ui/store/reducers/asideHeader.ts:49]: We need to use more common naming here:
`!DL.USER.isFederationUserOnCommunity` -> `DL.withNavigation` for example

File: src/ui/components/ErrorContent/ErrorContent.js [hunk 2]
```diff
@@ -128,6 +129,14 @@ class ErrorContent extends React.PureComponent {
             return null;
         }
 
+        if (isHtmlInDescription)
```
Comment (imsitnikov) [inline src/ui/components/ErrorContent/ErrorContent.js:132]: Add brackets plz

File: src/shared/types/common.ts [hunk 3]
```diff
@@ -129,6 +130,7 @@ export interface DLUser extends DLUserAccount {
     isFederationUser?: boolean;
     isGlobalFederationUser?: boolean;
     isLocalFederationUser?: boolean;
+    isFederationUserOnCommunity?: boolean;
```
Comment (imsitnikov) [inline src/shared/types/common.ts:133]: `withNavigation`

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/ui/store/reducers/asideHeader.ts (hunk 1 above)
Comment (Sergey-weber) [reply src/ui/store/reducers/asideHeader.ts]: added DL.HIDE_NAVIGATION

File: src/shared/types/common.ts (hunk 3 above)
Comment (Sergey-weber) [reply src/shared/types/common.ts]: added HIDE_NAVIGATION, by default will be false

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**